Gems/Atom/Utils/Code/Source/ImageComparison.cpp

```diff
@@ -9,11 +9,55 @@
 #include <Atom/Utils/ImageComparison.h>
 
 #include <AzCore/Casting/numeric_cast.h>
+#include <AzCore/Preprocessor/EnumReflectUtils.h>
 
 namespace AZ
 {
     namespace Utils
     {
+        AZ_ENUM_DEFINE_REFLECT_UTILITIES(ImageDiffResultCode);
+
+        void ImageDiffResult::Reflect(ReflectContext* context)
+        {
+            if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
+            {
+                ImageDiffResultCodeReflect(*serializeContext);
```

Comment:
There is an equivalent function to call for the behavior context so you don't have to reflect the enum manually.